### PR TITLE
add readme for db initialisation

### PR DESCRIPTION
**Pull request by ElevnLi**

### CODE REVIEW

File: search/README.md [hunk 1]
```diff
@@ -20,4 +20,16 @@ DJANGO_SECRET_KEY=
 POSTGRES_USER=
 POSTGRES_PASSWORD=
 POSTGRES_DB=
-```
\ No newline at end of file
+```
+
+### Initialise database, migrate and create superuser
+- Database init
+	1. Start PostgreSQL server: `docker-compose -f docker-compose-dev.yml up db`
```
Comment (jnothman) [inline search/README.md:27]:
```suggestion
	1. Start PostgreSQL server: `docker-compose -f docker-compose-dev.yml up -d db`
```
Make it launch in the background?

File: search/README.md [hunk 2]
```diff
@@ -20,4 +20,16 @@ DJANGO_SECRET_KEY=
...
+	2. Enter server container: `docker exec -it db bash`
+	3. Log in with the credential stored in `.env` file: `psql -U <POSTGRES_USER>`
```
Comment (jnothman) [inline search/README.md:29]: This isn't working for me. I have a .env with `POSTGRES_USER=me`. I'm getting:
```
bash-5.0# psql -U me
psql: error: could not connect to server: FATAL:  database "me" does not exist
```
This doesn't seem to be what I should expect from https://hub.docker.com/_/postgres. Any idea what's going on?

**Review by jnothman — COMMENTED** [review]
Sorry, this isn't yet working for me, which holds up my testing of other PRs :(

File: search/README.md (hunk 2 above)
Comment (ElevnLi) [reply search/README.md:29]: Try `psql` or `psql -U me postgres`

**Pull request merged**